IM_calculation/scripts/calculate_ims.py

```diff
@@ -109,11 +109,11 @@ def main():
         "-c",
         "--components",
         nargs="+",
-        choices=calc.COMPONENTS,
-        default=calc.COMPONENTS,
+        choices=list(Components.iterate_str_values()),
+        default=list(Components.iterate_str_values())[:4],
```

Comment:
Hmmm, we should probably update the default to just geom. I don't see a use case for all 4 at the moment.